core/src/labelled.rs

```diff
@@ -723,15 +748,66 @@ impl<Key, SourceValue> Transmogrifier<SourceValue, IdentityTransMog> for Field<K
     }
 }
 
-/// Implementation of `Transmogrifier` that maps over a `Vec` in a `Field`, transmogrifying the
-/// elements on the way past.
+macro_rules! transmogrify_seq {
+    ($container:ident) => {
+        /// Implementation of `Transmogrifier` that maps over a `$container` in a `Field`, transmogrifying the
+        /// elements on the way past.
+        impl<Key, Source, Target, InnerIndices>
+            Transmogrifier<$container<Target>, MappingIndicesWrapper<InnerIndices>>
+            for Field<Key, $container<Source>>
+        where
+            Source: Transmogrifier<Target, InnerIndices>,
+        {
+            fn transmogrify(self) -> $container<Target> {
+                self.value.into_iter().map(|e| e.transmogrify()).collect()
+            }
+        }
+    };
+}
+
+use std::collections::{LinkedList, VecDeque};
```

Comment:
Yikes, more things that will require a rebase of the `#[no_std]` PR.  I'd really rather push that through first.

Reply:
Fair point - what stage is that PR at?  I'm interested because at my company people really want to be able to use transmogrification in our projects but we're blocked behind needing the function added in this PR.

Is there anything I can do to help make progress on the `#[no_std]` PR to unblock this one?

Reply:
It is now merged